.github/helpers/generate-release-notes.sh

```diff
@@ -5,13 +5,7 @@ CHANGELOG=$(git --no-pager log --no-notes --no-decorate --oneline  v${1}...HEAD)
 
 ## Gather Framework version
 TESTCAFE_VER=$(< package-lock.json jq -r '.dependencies["testcafe"].version')
-NODEJS_VER=$(grep NODE_VERSION= Dockerfile | cut -d '=' -f 2)
-
-## Gather Browser versions
-BASE_IMAGE=$(grep FROM Dockerfile | cut -d ' ' -f 2)
-docker pull "${BASE_IMAGE}" > /dev/null 2>&1 || exit 1
-FF_VER=$(docker inspect ${BASE_IMAGE} | jq -r '.[0].ContainerConfig.Env | .[] | select(. | startswith("FF_VER="))' | cut -d '=' -f 2)
-CHROME_VER=$(docker inspect ${BASE_IMAGE} | jq -r '.[0].ContainerConfig.Env | .[] | select(. | startswith("CHROME_VER="))' | cut -d '=' -f 2)
+NODEJS_VER=$(grep NODE_VERSION: .github/workflows/release.yml | cut -d ':' -f 2 | tr -d ' ')
```

Comment:
See https://github.com/saucelabs/sauce-playwright-runner/pull/211#discussion_r1178416775. Looks like this repo still needs an `.nvmrc` file though.